maven-plugin/src/main/java/io/apicurio/codegen/maven/GenerateCodeMojo.java

```diff
@@ -0,0 +1,106 @@
+package io.apicurio.codegen.maven;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.Enumeration;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipException;
+import java.util.zip.ZipFile;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.maven.plugin.AbstractMojo;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+
+import io.apicurio.hub.api.codegen.JaxRsProjectSettings;
+import io.apicurio.hub.api.codegen.OpenApi2JaxRs;
+
+/**
+ * The main generate code mojo implementation.
+ * 
+ * @author eric.wittmann@gmail.com
+ */
+@Mojo(name = "generate")
+public class GenerateCodeMojo extends AbstractMojo {
+
+    @Parameter(required = true)
+    JaxRsProjectSettings projectSettings;
+
+    @Parameter(required = true)
+    File inputSpec;
+
+    @Parameter(defaultValue = "${project.build.directory}/generated-sources/jaxrs")
```

Comment:
Using `generated-sources/java` as default will avoid the extra folder inclusion when using the plugin, or you explicitly want to disambiguate it?

Reply:
nice 👍 I completely forgot about this 🤦